pandas/tests/io/json/conftest.py

```diff
@@ -7,3 +7,27 @@ def orient(request):
     Fixture for orients excluding the table format.
     """
     return request.param
+
+
+@pytest.fixture
+def json_dir_path(datapath):
+    """
+    The directory path to the data files needed for parser tests.
+    """
+    return datapath("io", "json", "data")
+
+
+@pytest.fixture(params=["ujson", "pyarrow"])
+def engine(request):
+    return request.param
```

Comment:
```suggestion
if request.param == "pyarrow"
    pytest.importorskip("pyarrow.json")
    return request.param
```

This will skip tests if pyarrow is not installed